uniflow/op/model/model_config.py

```diff
@@ -62,11 +62,12 @@ class NougatModelConfig(ModelConfig):
     batch_size: int = 1
     model_server: str = "NougatModelServer"
 
+
 @dataclass
-class BedrockModelConfig():
+class BedrockModelConfig:
     """Bedrock Model Config Class."""
 
-    aws_region: str 
+    aws_region: str = "us-west-2"
```

Comment:
nit: it is not idea to leave default region. You can consider to leave "" as default value if you really want.